tests/de.ovgu.featureide.fm.core-test/src/testFeatureModels/constraintDescriptionTwoRulesTest.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+	<featureModel>
+		<properties/>
+		<struct>
+			<and mandatory="true" name="Root">
+				<feature name="A"/>
+				<feature name="B"/>
+			</and>
+		</struct>
+		<constraints>
+			<rule>
+			<description>
+				Test Description
+			</description>
+				<conj>
+					<var>A</var>
+					<var>B</var>
+				</conj>
+			</rule>
+			<rule>
+			
+				<imp>
+					<var>A</var>
+					<var>B</var>
+				</imp>
+			<description>
+				Test Description
```

Comment:
I would use different descriptions to alse check that descriptions are not messed up. And maybe also a third rule without any description.